manifests/params.pp

```diff
@@ -441,7 +441,7 @@
   $server_idle_timeout                    = 1200000
   $server_web_idle_timeout                = 30000
   $server_connect_timeout                 = 120000
-  $server_enable_ruby_profiler            = false
+  $server_enable_ruby_profiler            = true
```

Comment:
yes, it's only an idea... otherwise the default value could also depend on the puppetversion...